src/win/thread.c

```diff
@@ -252,6 +252,9 @@ int uv_thread_getaffinity(uv_thread_t* tid,
   return 0;
 }
 
+int uv_thread_getcpu(void) {
+  return GetCurrentProcessorNumber();
```

Comment:
Should probably use GetCurrentProcessorNumberEx(), otherwise the return value can be misleading on systems with > 64 cores.

Reply:
We should not use GetCurrentProcessorNumberEx(), and the reason is below:
1)  By default, an application in windows is constrained to a single group(with at most 64 cores).
2)  The current libuv get/set affinity impl in Windows platform is constrained in a single group(with at most 64 cores),
     and the return value of uv_cpumask_size() in Windows is hardcoded to 64, which means that the maximum cpu 
     number seen from the libuv user is just 64.

Reply:
I believe Windows 10 removed that first restriction. And the second is a function, so it can be increased.

Reply:
For my second reason:  
My thread_getcpu() impl is compatiable with the current libuv implementation.
The existed libuv thread affinity in windows only supports inside the current processor group.
So uv_cpumask_size(), uv_thread_setaffinity(), uv_thread_getaffinity(), and related test code will need to be changed to support more than 64 cores. But this feature is another task.